src/app/Topology/Shared/utils.tsx

```diff
@@ -105,13 +105,26 @@ export const getStatusTargetNode = (node: TargetNode | EnvironmentNode): [NodeSt
           NodeStatus.warning,
           {
             title: 'Failed to compute JVM ID',
-            description: `Target ${node.target.alias} might be missing credentials.`,
+            description: (
+              <>
+                <Text component={TextVariants.p}>
+                  If target has JMX Authentication enabled, add the credential to Cryostat keyring.
+                </Text>
+                <Text component={TextVariants.p}>
+                  If the target has SSL enabled over JMX, add its certificate to Cryostat truststore. Restart is
+                  required.
```

Comment:
The Security view already says `Restart is needed to apply changes`, so I don't think it needs to be said here first. Maybe the text on the Security view should be updated to explain a little more, like `The Cryostat server must be restarted in order to reload the certificate store.`